planetaryimage/pds3image.py

```diff
@@ -152,18 +152,17 @@ def _save(self, file_to_write, overwrite):
 
         diff = 0
         if len(pvl.dumps(self.label, cls=encoder)) != label_sz:
-            diff = label_sz - len(pvl.dumps(self.label, cls=encoder))
+            diff = abs(label_sz - len(pvl.dumps(self.label, cls=encoder)))
```

Comment:
While converting image from type `int` to `float`, there was a condition where `label_sz` was 525 and `len(pvl.dumps(self.label, cls=encoder))` was 549. In this case, `diff` was `-24` which actually should've been `24`. 

Using `abs` resolves this issue.